### PR TITLE
[teleop] Complete Redesign of Science GUI

### PR DESCRIPTION
**Pull request by jnnanni**
All of the changes to the Science GUI made during the last school year. Changes are teleop-science-gui-improvments project. The changes in this PR are in the in progress column.

### CODE REVIEW

**Comment by CameronTressler** [comment]
Instead of creating a Google Doc with screenshots of LCMs, can you add a README.md file to base_station/gui/ and enumerate which LCM messages/channels are supposed to be sent and received (and say for each that you tested it). For now it can be specific to the Science GUI even though it's in the gui/ directory

**Review by CameronTressler — APPROVED** [review]
Readme will need to be updated with more detail in the future, but I think this should be sufficient for now

File: base_station/gui/src/components/SATask.vue [hunk 1]
```diff
@@ -49,11 +58,15 @@ import { mapGetters, mapMutations } from 'vuex'
 import Cameras from './Cameras.vue'
 import RoverMap from './RoverMap.vue'
 import CommIndicator from './CommIndicator.vue'
-import OdometryReading from './OdometryReading.vue'
+import OdometryReading from './OdometryReadingSA.vue'
```
Comment (cgiger00) [inline base_station/gui/src/components/SATask.vue:61]: do we need odom data in science?

Comment (cgiger00) [reply base_station/gui/src/components/SATask.vue:61]: we aren't planning for the gps to be plugged in at least during this

File: base_station/gui/src/static/rover_msgs.json [hunk 2]
```diff
@@ -890,6 +1018,7 @@
         "MicroCam",
         "Microscope",
         "Mosfet",
```
Comment (cgiger00) [inline base_station/gui/src/static/rover_msgs.json:1020]: can you get rid of this

**Review by cgiger00 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**